### ISSUE TITLE
stmp not working authentication failed

### ISSUE THREAD

**Issue by KyloJorgensen** [nodemailer/wildduck#95]
zone-mta[8390]: ERR! Plugins "Example Authentication" for "smtp:auth" failed with: Authentication failed

I found it

["example-auth"]
#enabled="receiver"
enabled=false
# only check authentication for interfaces with following names
#interfaces=["feeder"]

~
~
"/etc/zone-mta/plugins/example-auth.toml" 5L, 138C

**Comment by erdii** [comment]
this happened to me, too! Maybe a bug in the automated installation script?

**Comment by dramat1c** [comment]
yep this was the same issue i had..... works great now... thanks for posting this solution
